### ISSUE TITLE
Spelling fixes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8684]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)| @scop reported|
|-|-|
|Trac ID|trac#8684|
|Type|enhancement|
|Created|2016-07-26 10:51:53Z|

<details><summary>Searchable metadata</summary>

```
trac-id__8684 8684
type__enhancement enhancement
reporter__scop scop
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469530313105797 1469530313105797
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_1"></a>|@scop commented|
|-|-|

https://github.com/twisted/twisted/pull/417

**Comment by twisted-trac** [comment]
|<img alt="Ville Skyttä's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|Ville Skyttä commented|
|-|-|

In changeset 90ff7d6cfb2c2073c3ba0dcc37eacf1e4fb630fd
```
#!CommitTicketReference repository="" revision="90ff7d6cfb2c2073c3ba0dcc37eacf1e4fb630fd"
Add topfile for #8684
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 6bbaf9154ddb7f34f2795c89f8816a70bfe21cfa
```
#!CommitTicketReference repository="" revision="6bbaf9154ddb7f34f2795c89f8816a70bfe21cfa"
Merge scop/8684-scop-spelling: Spelling fixes.


Author: scop
Reviewer: Alex
Fixes: #8684
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
